ltclient/gulpfile.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const gulp = require('gulp');
+const svgstore = require('gulp-svgstore');
+const inject = require('gulp-inject');
+
+// const sassSrc = 'src/scss/**/*.scss';
+// const dest = 'dist';
+
+// gulp.task('sass', () => {
+//   return gulp.src(sassSrc)
+//     .pipe(sass({
+//       outputStyle: 'expanded'
+//     }).on('error', sass.logError))
+//     .pipe(autoprefixer())
+//     .pipe(gulp.dest(dest));
+// });
+
+// gulp.task('sass:watch', () => {
+//   gulp.watch(sassSrc, ['sass']);
+// });
+
+// gulp.task('sass:minify', () => {
+//   return gulp.src(sassSrc)
+//     .pipe(sass({outputStyle: 'compressed'}).on('error', sass.logError))
+//     .pipe(autoprefixer())
+//     .pipe(gulp.dest(dest));
+// });
+
+gulp.task('svg', () => {
```

Comment:
I see u are using Webpack already, do u agree to use one pack tools for all front-end stuff?